modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -573,11 +576,25 @@ private void setCurrentGroupTopology(RaftGroupConfiguration config) {
 
     @Override
     public void onSnapshotSave(Path path, Consumer<Throwable> doneClo) {
+        onSnapshotSaveHandler.onSnapshotSave(snapshotInfo(), List.of(this))
+                .whenComplete((unused, throwable) -> doneClo.accept(throwable));
+    }
+
+    private PartitionSnapshotInfo snapshotInfo() {
         long maxAppliedIndex = max(storage.lastAppliedIndex(), txStatePartitionStorage.lastAppliedIndex());
         long maxAppliedTerm = max(storage.lastAppliedTerm(), txStatePartitionStorage.lastAppliedTerm());
 
-        onSnapshotSaveHandler.onSnapshotSave(maxAppliedIndex, maxAppliedTerm, List.of(this))
-                .whenComplete((unused, throwable) -> doneClo.accept(throwable));
+        byte[] configuration = storage.getStorage().committedGroupConfiguration();
+
+        assert configuration != null : "Trying to create a snapshot without Raft group configuration";
+
+        return new PartitionSnapshotInfo(
+                maxAppliedIndex,
+                maxAppliedTerm,
+                storage.leaseInfo(),
+                configuration,
+                Set.of(storage.tableId())
```

Comment:
Could you please add javadoc to `PartitionSnapshotInfo#tableIds` that would explain what these tables are and why we need them in the snapshot information?